user_docs/en/userGuide.t2t

```diff
@@ -1590,6 +1592,15 @@ This can be useful if it is hard to distinguish between pronunciation of symbols
 
 The delayed character description will be cancelled if other text is spoken during that time, or if you press the ``control`` key.
 
+====Switch between the following speech modes: ====[SpeechModesDisabling]
+This checkable list allows to select which speech modes are included when cycling between them using ``NVDA+s``.
+Modes which are unchecked are excluded.
+By default all modes are included, also if any new mode is added to NVDA it will also be included by default.
+
+For example if you do not need to use "beeps" mode and the "on-demand" mode you should uncheck these two, and keep both "off" and "talk" checked.

```

Comment:
Could we change the example with "talk" and "on-demand" checked. This would be an opportunity to promote the new "on-demand" mode. This way people can realize that "talk" and "on-demand" is a completely valid use case, and probably even the one that the majority of people would need.

```suggestion
For example if you do not need to use "beeps" and "off" mode  mode you should uncheck these two, and keep both "talk" and "on-demand" checked.
```

Reply:
> Note that I would not have requested to promote this way the new "on-demand" mode if it was only a niche feature.

I'm not sure we can judge this right now. There were some requests for it to be added to NVDA, (and obviously you needed this mode for your daily usage), but it hadn't seen much use outside of Alpha users, and we do not actually know if it would be popular, or perhaps would have to be redefined to be more useful.

> 
> Maybe I should have added something more explicit on this use case in the dedicated paragraph of the user guide.

The user guide paragraph is pretty clear with regard to how the mode works, and when it can be potentially useful, thanks for writing it! I'm not partially sighted, nor doing any audio production, so I just cannot comment on these use cases. For me personally the 'on-demand' mode as implemented today is certainly not useful in the calls/meetings situations, as it offers way too little feedback, and in general I'm content with either having full speech enabled in such cases, or relying on a Braille display. To expand on this a little, when I am an active participant of the meeting i.e. screen sharing, or being an active speaker (where I for instance need live regions announcements to know that someone raised their hand),  I need the full speech. When I am a passive participant then I am either performing the steps described by the meeting's host (again full speech is necessary in such cases), or at the very least reading slides or other supporting  materials. This technically can be done in the on-demand mode, but since it limits me to reading with review cursor full speech is just easier. Note that most meetings I had were in an academic context where I participated as a student in lectures or labs, so perhaps that makes a difference.

> 
> I have given a big explanation, but feel free to accept or decline my suggestion on this point.

I'd like to wait for more feedback here. Also if we will decide to uninstall NoBeepsSpeechMode automatically and reconfigure NVDA so that its behavior is preserved,  I'll certainly apply your suggestion, as in that case most users would get the behavior they're used to, and users looking for an example should be given a chance to try the new mode.


> 
> As an aside, I am curious to know if you understand why the on-demande mode can be an important use case for non-braille users, and if you think that a clarification should be added in the User Guide on this use case (can be done in this PR or in a separate one). Thanks for your feedback.

I see no use case for myself, and also have no idea if the documentation needs any clarification, as it is quite well written, I simply disagree with some of its conclusions.

Reply:
I'm convinced. It makes sense to improve visibility of the 'on-demand' mode, so that we can get feedback as to its usefulness.